pkg/kf/commands/utils/prefix_writer.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+)
+
+type prefixWriter struct {
+	nw newlineWriter
+}
+
+func NewPrefixWriter(prefix string, dest io.Writer) io.Writer {
+	return &prefixWriter{
+		nw: newlineWriter{
+			dest: func(data []byte) (int, error) {
+				n, err := fmt.Fprintf(dest, "%s%s\n", prefix, data)
+				if err != nil {
+					return 0, err
+				}
+				return n - len(prefix), nil
+			},
+		},
+	}
+}
+
+func (w *prefixWriter) Write(data []byte) (int, error) {
+	return w.nw.Write(data)
+}
+
+type newlineWriter struct {
```

Comment:
I think it makes sense to stick these in an internal package like `kfioutil` so we don't end up with a giant utils package.

Reply:
What full package name were you thinking?